misc/iitthesis-extra.sty

```diff
@@ -42,10 +42,10 @@
 %%
 
 \def\filename{iitthesis-extra}
-\def\fileversion{v0.3}
-\def\filedate{2012/03/24}
```

Comment:
This should definitely be updated, and I've neglected to do so for quite a long time... thanks for suggesting it.